### DISCUSSION TITLE
how to check the health of the agent

### DISCUSSION THREAD

**Discussion by tradexsrl** [netdata/netdata#17987] [Q&A] [answered]
Hi , on windows prometheus exporter there is a way to check if the "agent" is running by checking via a json query http://IP:9182/health
I do it easily in uptimekuma , so i get notification if something is wrong.
i need to know what endpoint i need to query on agent installed on linux machine and in docker .
thank's

**Comment by ilyam8** [comment]
You can use `/api/v1/info`

**Reply by tradexsrl** [reply, marked as answer]
thank's it works
